docs/courses/README.md

```diff
@@ -1,28 +1,104 @@
 # Contributing to LibreLingo courses
 
-LibreLingo courses consist of files that contain course content. These files
-are organized in a specific directory structure.
-[Learn how to edit or create courses](editing-courses.md)!
+This document describes how LibreLingo's course system works and how to contribute to course content.
 
+Understanding this document is the first step if you want to:
+- edit or improve an existing course
+- contribute a new accepted answer
+- create a new course from scratch
+
+This is also a good resource if you just want to learn about the course system in general.
+
+---
+
+**NOTE**: If you are not clear about anything on this document feel free to post on [Discussions](https://github.com/kantord/LibreLingo/discussions) so that the community can help you further.
+
+---
+
+**Table of Contents:**
+- [Getting started](#getting-started)
+- [Basics](#basics)
+  - [Terminology](#terminology)
+  - [Exploring the courses](#exploring-courses)
+  - [Things to do for new contributors](#things-new-contributors)
+- [Courses currently open for contributions](#courses-open)
+  - [For English speakers](#courses-from-english)
+- [Caution](#caution)
+  - [Languages that don't use the Latin alphabet](#non-latin-alphabet)
+
+## Getting started
+
+- Totally new to LibreLingo? Head to the [Basics](#basics) section! It will give you a general understanding about things.
```

Comment:
```suggestion
- Totally new to LibreLingo? Head to the [Basics](#basics) section! It will give you a general understanding of the course system.
```

`of` sounds more natural + better to be specific than use vague words like `things`